resources/views/welcome.blade.php

```diff
@@ -1,17 +1,9 @@
 @extends('layouts/app')
 @section('content')
 <div class="container">
-  <ul class="nav nav-tabs col-md-offset-1 col-md-9">
-    <li class="active"><a href="#tab1" data-toggle="tab">Products</a></li>
-    <li><a href="#tab2" data-toggle="tab">Orders</a></li>
-  </ul>
-  <div class="tab-content col-md-offset-1 col-md-9" id="TabContents">
-    <div class="tab-pane fade active in" id="tab1">
-      <a class="btn btn-primary margin-10" type="button" href"/products/create">Add Product</a>
-      @include("partials/products", ["products" => App\Product::all(), "dashboard" => true])
-    </div>
-    <div class="tab-pane fade" id="tab2">
-      @include("partials/orders", ["orders" => App\Order::all()])
-    </div>
+  <div class="jumbotron">
+    <h1>Welcome to the website!</h4>
+    <p>The Wordpress page that Mr. Beretta has made would idealy be placed here.</p>
```

Comment:
Probably replace with something like "Please log in or register" followed by buttons for doing so.